07-custom-jobs.Rmd

```diff
@@ -0,0 +1,79 @@
+
+</details>
+
+You can learn more about other use cases of `sbatch` at the Biomedical Data Science Wiki's page on [computing jobs](https://sciwiki.fredhutch.org/scicomputing/compute_jobs/#submitting-jobs).
+
+### Saving your `sbatch` options
+
+The options you pick for the `sbatch` command, especially around CPU and RAM requests, can have enormous impact on your computing speed and quality. Therefore, it is often a good idea to save the options you have picked into the shell script itself. Suppose that you launched your job like this:
+
+```         
+sbatch -c 4 01.sh
+```
+
+To save the `-c 4` option in your shell script, you can add the following header `#SBATCH -c 4` to your script:
+
+```         
+#!/bin/bash
+
+#SBATCH -c 4
+
+echo "Hello, World!"
+```
+
+Now, you will automatically request 4 CPUs when you run:
+
+```         
+sbatch 01.sh
+```
+
+## Monitoring and Ending Jobs
+
+Often, our jobs take time to run, and we want to see the status of our jobs. You can examine the status of your job via the `squeue` command. The default option will show *all* jobs running on the cluster, including other users, so it's common to use the `-u` option to specify the user. The example user here is `clo2`:
+
+```         
+squeue -u clo2
+             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
+          36277207 campus-ne HelloWor     clo2 PD       0:00      1 (Priority)
+```
+
+Here, the status ("ST") shows `PD`, which means that the job hasn't started to run yet. When it is running the status will be `R`.
+
+Some common reasons for `PD` include: there are no available resources left for the requested resources for your job, or that your account has already maxed out its allocated resources. The decision of which user's jobs are running is based on the Slurm scheduler system. You can learn more about `squeue`'s outputs at the Biomedical Data Science wiki's [page here](https://sciwiki.fredhutch.org/scicomputing/compute_jobs/#why-isnt-my-job-running).
+
+If you want to end a job that is either waiting (`PD` status) or running, you can enter:
+
+```         
+scancel [job id]
+```
+
+where `[job id]` is a numerical ID attached to your unique job. It can be found under `JOBID` when you run `squeue`.
```

Comment:
Could we add a dictionary <div> summary at the bottom similar to the other pages?